.gitignore

```diff
@@ -119,3 +119,5 @@ build
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+packages/ui/tsconfig.tsbuildinfo
```

Comment:
It's probably better to ignore this file regardless of the folder
```ssh
*.tsbuildinfo
```

Reply:
agreed